components/map/layers/marker.md

```diff
@@ -81,20 +90,105 @@ The following example demonstrates how to configure the Map Marker Layer.
 }
 ````
 
->caption The result from the above code snippet.
+## Marker Template
+
+To customize the marker appearance, set the `Template` parameter in the corresponding marker inner tag - `MapLayerMarkerSettings`. It accepts `string` and `RenderFragment`.
+
+The general syntax of the template is based on the [Kendo Templates](https://docs.telerik.com/kendo-ui/framework/templates/overview).
 
-![](../images/marker-layer.png)
+>caption Custom Markers.
+
+````CSHTML
+@* This code snippet showcases an example of customizing the Marker appearance. *@
+
+<TelerikMap Center="@Center"
+            Zoom="3">
+    <MapLayers>
+        <MapLayer Type="@MapLayersType.Marker"
+                  Data="@MarkerData1"
+                  LocationField="@nameof(MarkerModel.LatLng)"
+                  TitleField="@nameof(MarkerModel.Title)">
+                  <MapLayerMarkerSettings Template="<span class='custom-marker-class'>#= dataItem.Title #</span>">
+                  </MapLayerMarkerSettings>
+        </MapLayer>
+
+        <MapLayer Type="@MapLayersType.Tile"
+                  Attribution="@Attribution"
+                  Subdomains="@Subdomains"
+                  UrlTemplate="@UrlTemplate">
+        </MapLayer>
+
+        <MapLayer Type="@MapLayersType.Marker"
+                  Data="@MarkerData2"
+                  LocationField="@nameof(MarkerModel.LatLng)"
+                  TitleField="@nameof(MarkerModel.Title)">
+                  <MapLayerMarkerSettings Template="#= dataItem.LatLng #">
+                      <MapLayerMarkerSettingsTooltip>
+                          <Template>
```

Comment:
We can split the examples into 2 sections - one for marker template and one for marker tooltip template.

Reply:
I will just remove the marker tooltip template, as there is already a sample on that page that shows it.